### DISCUSSION TITLE
Node Server Stop

### DISCUSSION THREAD

**Discussion by travisfont** [requarks/wiki#7042] [Error / Bug Report] [answered]
Whenever I close the terminal window, the node server dies.

I just updated: https://docs.requarks.io/en/install/upgrade
Using "Linux" instructions, and all steps besides 7 `node server` doesn't seem efficient enough.
Without any errors either.

**Update,** I was able to get the node server to run using `sudo npm install pm2@latest -g` and `pm2 start server` but this not what's documented...

**Comment by NGPixel** [comment, marked as answer]
The instructions to run as a service are well documented in each platform guide... .e.g. for Linux:

https://docs.requarks.io/install/linux#run-as-service

The upgrade instructions focus solely on the upgrade part. There are dozens of ways to run wiki.js unattended and the upgrade page is not the place to cover that.

**Reply by travisfont** [reply]
Ah, this makes sense, sweet beans.
This was previously done, but the information on this service existing was lost (for myself).
Not sure why the enable on boot didn't work, unless I purposely didn't enable it previously.

May I update the upgrade docs to contain your reference to the 'run as service' along with the pm2 alternative?
